tests/unit/test_lockfile.py

```diff
@@ -49,7 +52,8 @@ def test_stage_overwrites_if_already_exists(tmp_dir, dvc):
     stage = PipelineStage(name="s2", repo=dvc, path="path", cmd="command3")
     lockfile.dump(stage)
     assert lockfile.load() == {
-        "s2": {"cmd": "command3"},
+        "stages": {"s2": {"cmd": "command3"}},
+        "meta": {"version": "2.0"},
```

Comment:
Maybe let's just have top-level `schema: 2.0`? Meta is confusing with `meta` from dvc.yaml and `version` is confusing with dvc version.